.github/workflows/main.yml

```diff
@@ -22,7 +22,7 @@ jobs:
         run: composer install --prefer-dist --no-progress
 
       - name: Run PHP Code Sniffer
-        run: vendor/bin/phpcs .
+        run: vendor/bin/phpcs . --ignore=.../Resources/public/administration/css/adyen-payment-shopware6.css
```

Comment:
@Morerice could you try using `phpcs:disable` or a specific `phpcs:ignore ...` command in this css file?

Reply:
Updated in d156d16